src/groups_v2/model.rs

```diff
@@ -105,31 +133,31 @@ pub enum GroupChange {
         #[derivative(Debug = "ignore")]
         profile_key: ProfileKey,
     },
-    // for open groups
     NewPendingMember(PendingMember),
     DeletePendingMember(Uuid),
     PromotePendingMember {
         uuid: Uuid,
```

Comment:
Are pending members `Uuid`s, `ServiceId`s or `Aci`s? Maybe it's worth to comment the code whenever `Uuid` or `ServiceId` is used, to document *why* that specific form is used.

E.g., for invites, IIRC it's a `ServiceId`, because members can be invited based on their phone number (iirc).

Reply:
Pending member is ServiceId indeed.